pytest.ini

```diff
@@ -2,11 +2,13 @@
 # only the files with prefix `pytest_` would be recognized as pytest test scripts.
 python_files = pytest_*.py
 
+# ignore PytestExperimentalApiWarning for record_xml_attribute
+# set traceback to "short" to prevent the overwhelming tracebacks
 addopts =
   -s
   --embedded-services esp,idf
-  -W ignore::_pytest.warning_types.PytestExperimentalApiWarning
   --tb short
+  --skip-check-coredump y
```

Comment:
I wonder why do we need to skip coredump check. Does it mean that the test app produce any? Is it possible to miss a crash with this settings?